modules/extensions/kubeconfig.tf

```diff
@@ -5,7 +5,21 @@ data "oci_containerengine_cluster_kube_config" "kube_config" {
   cluster_id = var.cluster_id
 }
 
+# Terraform doesn't support conditional dynamic blocks for lifecycle, so resource
+# is repeated with and without it based on update_kubeconfig boolean variable
+# Not ideal, but mitigates reported change on every apply
 resource "local_file" "kube_config_file" {
```

Comment:
As mentioned above, I was actually thinking of getting rid of this. The problem is that this file is overwritten if I use the oke module in a multi cluster scenario. It's also a remnant from when we had a public cluster only. Granted we can still deploy public cluster and users may still want this as a convenience. 

1. What's your view on keeping/not keeping?
2. If we keep, can we please rename by cluster name instead?

Reply:
We agreed to remove this.

Reply:
We decided to leave for another PR.